localstack/services/cloudformation/models/__init__.py

```diff
@@ -25,4 +25,5 @@
     "sqs",
     "ssm",
     "stepfunctions",
+    "stores",
```

Comment:
CloudFormation is a bit of a special case - the `models` package is more intended for the CFn resource model implementations (e.g., `AWS::Lambda::Function`, etc). 

I understand that the name clashes with the general convention of putting stores into `models.py` - but perhaps we could move the file to `services/cloudformation/stores.py`?

Reply:
Thanks, I wasn't sure if the old path was appropriate.

Fixed in 36957feb